alacritty_terminal/src/keyboard_motion.rs

```diff
@@ -0,0 +1,767 @@
+use std::cmp::min;
+
+use serde::Deserialize;
+
+use crate::event::EventListener;
+use crate::grid::{GridCell, Scroll};
+use crate::index::{Column, Point};
+use crate::term::cell::Flags;
+use crate::term::{Search, Term};
+
+/// Possible keyboard motion movements.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Deserialize)]
+pub enum KeyboardMotion {
+    /// Move up.
+    Up,
+    /// Move down.
+    Down,
+    /// Move left.
+    Left,
+    /// Move right.
+    Right,
+    /// Move to start of line.
```

Comment:
I feel like it should be:

```suggestion
    /// Move to the start of the line.
```

Same should apply to most other comments in this enum and other places in this file, but I'm not sure about this change.

Reply:
I think it's fine without, still reads nicely and isn't really incorrect either.

Especially the double `the`s would kinda make the comments annoying to read otherwise, so I've chosen to keep it shorter here.